docs/common_workflows.md

```diff
@@ -0,0 +1,47 @@
+# Getting the development environment set up
+
+## What your directory should look like
+
+```text
+<root>
+├── development-environment
+├── <Any other v3 repos you may be working with>
+└── v3-website
+```
+
+## Git workflow to working from a fork
+
+### Cloning `exercism/v3-website` and maintaining your `fork`
+
+```text
+> cd <root>
+> git clone https://github.com/exercism/v3-website.git
+> cd v3-website
+```
+
+After a pull request is open against `exercism/v3-website`, the remote fork's branch can be checked out [following this guide][github-mod-inactive-pr-local].  After making a commit, you can't `git push` to the `exercism`'s repo, but you can to your own fork
+
+```text
+> git push <remote-name> HEAD:<remote-branch-name>
+```
+
+### Using your own fork
+
+```text
+> cd <root>
+> git clone <your v3-website fork> v3-website 
+```
+
+Then when making commits you can easily push the commit to the origin
+
+## Running tests
+
+Now when you start the development environment, it will use your `v3-website` repo as the source for the rails application. When the development environment is running, enter the website's running docker container to run tests.
+
+```text
+> docker exec -it development-environment_website_1 bash
```

Comment:
There is an easier (and more foolproof) way:

```suggestion
> ./bin/shell website
```

Reply:
I like it! Maybe I would first use the short way you suggest, then explain what that does so people can understand the magic.

Reply:
```suggestion
> ./bin/shell website

# This is a short cut for: 
# > docker exec -it development-environment_website_1 bash
```